src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Should wildcards be used here?
```suggestion
import static org.junit.jupiter.api.Assertions.*;
```